src/ruuvi.firmware.c.emProject

```diff
@@ -355,7 +355,6 @@
       <file file_name="../nRF5_SDK_15.3.0_59ac345/modules/nrfx/drivers/src/nrfx_twi.c" />
       <file file_name="../nRF5_SDK_15.3.0_59ac345/modules/nrfx/hal/nrf_nvmc.c" />
       <file file_name="../nRF5_SDK_15.3.0_59ac345/integration/nrfx/legacy/nrf_drv_twi.c" />
-      <file file_name="../nRF5_SDK_15.3.0_59ac345/modules/nrfx/drivers/src/nrfx_nfct.c" />
```

Comment:
PCA10040 has NFC, why the removal?

Reply:
The compiler crashed with error:
`The object file Output/Debug/Obj/pca10040/nrfx_nfct.o has been supplied more than once`

Reply:
Ah yes, this is probably related to https://github.com/ruuvi/ruuvi.firmware.c/issues/162 where SDK15.3 NFC was overridden with SDK17.2 NFC